lib/uplinkc/uplink.go

```diff
@@ -19,24 +19,24 @@ type Uplink struct {
 // an error in cerr, when there is one.
 //
 // Caller must call close_uplink to close associated resources.
-func new_uplink(cfg C.UplinkConfig, cerr **C.char) C.Uplink {
+func new_uplink(cfg C.UplinkConfig_t, cerr **C.char) C.UplinkRef_t {
 	scope := rootScope("inmemory") // TODO: pass in as argument
 
 	libcfg := &uplink.Config{} // TODO: figure out a better name
-	libcfg.Volatile.TLS.SkipPeerCAWhitelist = cfg.Volatile.TLS.SkipPeerCAWhitelist == 1
+	libcfg.Volatile.TLS.SkipPeerCAWhitelist = cfg.Volatile.TLS.SkipPeerCAWhitelist == C.bool(true)
```

Comment:
Is casting `true` to `C.bool` the correct behavior?

Reply:
I believe so, we changed the type of `SkipPeerCAWhitelist` to be a boolean.